compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -47,6 +47,33 @@ def AMDAIEFuseFillIntoForall :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEFuseFillIntoForallPass()";
 }
 
+def AMDAIEInsertLoopsForVectorization :
+    Pass<"iree-amdaie-insert-loops-for-vectorization", "func::FuncOp"> {
+
+    let summary = "Transforms linalg.generic operations with matmul-like semantics.";
```

Comment:
Add more descriptions for summary. "Transforms .... to replace outer-dimension with scf.for loops.